lib/browser/lighthouse.js

```diff
@@ -33,8 +33,9 @@
 'use strict';
 
 const assert = require('assert');
-const isEmpty = require('lodash/isEmpty');
-const flatMap = require('lodash/flatMap');
+const flatMap = require('array.prototype.flatmap');
+
+flatMap.shim(); // TODO: remove with node 12
```

Comment:
I would recommend using it directly as a ponyfill - it's safer and not that much harder to refactor later.